src/euidSdk.ts

```diff
@@ -0,0 +1,49 @@
+import { EventType, Uid2CallbackHandler } from './uid2CallbackManager';
+import { CallbackContainer, ProductDetails, UID2SdkBase, UID2Setup } from './uid2Sdk';
+
+export class EUID extends UID2SdkBase {
+  // Deprecated. Integrators should never access the cookie directly!
+  static get COOKIE_NAME() {
```

Comment:
If this is deprecated, is there a way to call it out more directly if someone is using it?  Maybe a console statement?

Reply:
Nice idea.